tests/jest/components/tab/SearchInput.spec.js

```diff
@@ -87,30 +92,27 @@ describe('SearchInput.vue tests', () => {
 				await wrapper.setData({
 					state: STATE.LOADING,
 				})
-
-				await inputField.setValue('org')
+				await inputField.setValue('a')
+				await inputField.setValue('')
 
 				expect(wrapper.vm.searchResults).toMatchObject([])
 				expect(wrapper.vm.state).toBe(STATE.OK)
+				axiosSpy.mockRestore()
 			})
 			it.each([
 				{
-					search: 'o',
-					expectedCallCount: 0,
-				},
-				{
-					search: 'or',
+					search: '',
 					expectedCallCount: 0,
 				},
 				{
-					search: 'org',
-					expectedCallCount: 0,
+					search: 'o',
+					expectedCallCount: 1,
 				},
 				{
-					search: 'orga',
+					search: 'or',
 					expectedCallCount: 1,
 				},
-			])('should send search request only if the search text is greater than search threshold', async ({
+			])('should send search request only if the search text is >= than search char limit', async ({
```

Comment:
```suggestion
			])('should send search request only if the search text is greater than or equal to the search char limit', async ({
```
IMO, this will be more readable.